src/count_occurrence.md

```diff
@@ -0,0 +1,77 @@
+# count_occurrence
+To Count The Occurrence of value in an array
+
+## Result
+
+```Py
+   count_occurrences([2,2,4,5,6], target_value) // '2'
+```
+
+```JavaScript
+   count_occurrences(myArray, targetValue);
+```
+
+```C++
+   count_ocurrences({4,5,8,6,8,1,8}, 8); // '3'
+```
+
+```PHP
+   count_ocurrences({4,5,8,6,8,1,8}, 8); // '3'
+```
```

Comment:
Remove all extra ones, not needed, follow the [template.md](https://github.com/rahulrk-dev/snips/blob/main/template.md) file